.github/workflows/build_wheels_linux.yml

```diff
@@ -23,8 +23,8 @@ jobs:
       matrix:
         include:
           - repository: pytorch/data
-            pre-script: ""
-            post-script: ""
+            pre-script: packaging/pre_build_script_linux.sh
+            post-script: packaging/post_build_script_linux.sh
```

Comment:
Right now, the `post-script` is executed after uploading to github. Should we reverse the order in `pytorch/test-infra/.github/workflows/build_wheels_linux.yml@main`?

Reply:
I guess no. To be able to run the post-script, we [need the binaries to be](https://github.com/pytorch/data/pull/930/files#diff-effc02c0a703f8d729451b33e4c6fc9f1c9d7012fefd34e2e70cd259019acc11R6) in the `dist` folder.

Reply:
Understood. Then, in order to respect the path, we can technically remove the binaries from `dist` and move the repaired binaries from `wheelhouse` back to `dist`

Reply:
Then `post-script` is basically doing nothing, right? All uploaded binaries remain the same as non-manylinux compatible.